pages/treeherder.py

```diff
@@ -31,13 +31,13 @@ class TreeherderPage(Base):
     _quick_filter_locator = (By.ID, 'quick-filter')
     _repos_menu_locator = (By.ID, 'repoLabel')
     _result_sets_locator = (By.CSS_SELECTOR, '.result-set:not(.row)')
+    _resultset_visible_locator = (By.CSS_SELECTOR, 'span.hidden')
     _unchecked_repos_links_locator = (By.CSS_SELECTOR, '#repoLabel + .dropdown-menu .dropdown-checkbox:not([checked]) + .dropdown-link')
     _unclassified_failure_count_locator = (By.ID, 'unclassified-failure-count')
     _unclassified_failure_filter_locator = (By.CSS_SELECTOR, '.btn-unclassified-failures')
 
     def wait_for_page_to_load(self):
-        self.wait.until(lambda s: self.unclassified_failure_count > 0)
-        return self
+        self.wait.until(lambda s: len(self.result_sets) >= 1)
```

Comment:
This may return when one result set exists, even if more are yet to load. If that's okay, then r+ but I think it would be better if we could tell when the page has finished the initial load of results.